tools/clang/unittests/HLSLExec/LongVectorOps.def

```diff
@@ -52,6 +53,12 @@ OP_DEFAULT_DEFINES(Bitwise, Not, 1, "TestUnaryOperator", "~",
                    " -DFUNC_UNARY_OPERATOR=1")
 OP(Bitwise, LeftShift, 2, "", "<<", "", Default1, BitShiftRhs, Default3)
 OP(Bitwise, RightShift, 2, "", ">>", "", Default1, BitShiftRhs, Default3)
+OP_DEFAULT(Bitwise, Saturate, 1, "saturate", "")
+OP_DEFAULT(Bitwise, ReverseBits, 1, "reversebits", "")
+OP(Bitwise, CountBits, 1, "countbits", "", "", Bitwise, Default1, Default2)
+OP(Bitwise, FirstBitHigh, 1, "firstbithigh", "", "", Bitwise, Default1,
+   Default2)
+OP(Bitwise, FirstBitLow, 1, "firstbitlow", "", "", Bitwise, Default1, Default2)
```

Comment:
Not sure if it matters, but I'm a little suprised these aren't of this format:

```
OP(Bitwise, CountBits, 1, "countbits", "", "", Bitwise, Default2, Default3)
```

(ie Bitwise just replaces Default1)

Reply:
Co-pilot autocomplete accident.